src/load.jl

```diff
@@ -0,0 +1,24 @@
+module Load
```

Comment:
The name of the module is up to debate, I'm not attached to it :slightly_smiling_face:

Reply:
I would probably call it what it is: a `UnixDlopenHack` module, or, if that is too casual, maybe `UnixForceDlopen`. Just `Load` is IMHO really too generic.